blog/zh-cn/dubbo-27-features.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Dubbo2.7 三大新特性详解
+keywords: Dubbo2.7
+description: 异步化改造,三大中心改造,服务治理增强
+---
+
+# 
+
+## 1 背景介绍
+
+自 2017 年 7 月阿里重启 Dubbo 开源，到目前为止 github star 数，contributor 数都有了非常大的提升。2018 年 2 月 9 日阿里决定将 Dubbo 项目贡献给 Apache，经过一周的投票，顺利成为了 Apache 的孵化项目，也就是大家现在看到的 **Incubator Dubbo**。预计在 2019 年 4 月，Dubbo 可以达成毕业，成为 Apache 的顶级项目。
+
+## 2 分支介绍
+
+![分支](http://kirito.iocoder.cn/image-20190321153455566.png)
+
+Dubbo 目前有如图所示的 5 个分支，其中 2.7.1-release 只是一个临时分支，忽略不计，对其他 4 个分支进行介绍。
+
+- 2.5.x 近期已经通过投票，Dubbo 社区即将停止对其的维护。
+- 2.6.x 为长期支持的版本，也是 Dubbo 贡献给 Apache 之前的版本，其包名前缀为：com.alibaba，JDK 版本对应 1.6。
+- 3.x-dev 是前瞻性的版本，对 Dubbo 进行一些高级特性的补充，如支持 rx 特性。
+- master 为长期支持的版本，版本号为 2.7.x，也是 Dubbo 贡献给 Apache 的开发版本，其包名前缀为：org.apache，JDK 版本对应 1.8。
+
+> 如果想要研究 Dubbo 的源码，建议直接浏览 master 分支。
+
+## 3 Dubbo 2.7 新特性
+
+Dubbo 2.7.x 作为 Apache 的孵化版本，除了代码优化之外，还新增了许多重磅的新特性，本文将会介绍其中最典型的三个新特性：
+
+- 异步化改造
+- 三大中心改造
+- 服务治理增强
+
+## 4 异步化改造
+
+### 4.1 几种调用方式
+
+![调用方式](http://kirito.iocoder.cn/image-20190321160844133.png)
```

Comment:
I suggest putting the pictures in the project directory

Reply:
My other article also relies on my own domain.

Reply:
ok

Reply:
I am +1 on putting the pictures in the project directory. Actually shall we really need this picture?